cas-server-support-osf/src/main/java/io/cos/cas/adaptors/postgres/daos/OpenScienceFrameworkDaoImpl.java

```diff
@@ -135,7 +135,8 @@ public OpenScienceFrameworkInstitution findOneInstitutionById(final String id) {
     public List<OpenScienceFrameworkInstitution> findAllInstitutions() {
         try {
             final TypedQuery query = entityManager.createQuery(
-                    "select i from OpenScienceFrameworkInstitution i where i.loginUrl is not null and i.deleted = false",
+                    "select i from OpenScienceFrameworkInstitution i "
+                            + "where i.delegationProtocol is not empty and i.deleted = false",
```

Comment:
`cas-pac4j` based institution login (e.g. okstate) has `null` or empty `loginUrl`. Query for `delegationProtocol` instead.